crates/aptos/src/common/types.rs

```diff
@@ -1157,14 +1157,8 @@ pub struct MovePackageDir {
 
     /// ...or --bytecode BYTECODE_VERSION
     /// Specify the version of the bytecode the compiler is going to emit.
-    /// Defaults to `6`, or `7` if language version 2 is selected
-    /// (through `--move-2` or `--language-version=2`), .
-    #[clap(
-        long,
-        default_value_if("move_2", "true", "7"),
-        alias = "bytecode",
-        verbatim_doc_comment
-    )]
+    /// Defaults to `7`.
```

Comment:
The default is set programmatically based on language versions, for lang version 2.2, it will likely be set to 8.

Reply:
Could you explain more on this? It should not be set to 8 when the language version is updated to 2.2

Reply:
See comment: https://github.com/aptos-labs/aptos-core/blob/main/third_party/move/move-model/src/metadata.rs#L276

I think the next language version (2.2) will require bytecode version update (due to closure support), which is what I inferred from that comment.

Reply:
Oh, in that case, it is fine